etl/steps/data/garden/state_capacity/2023-10-19/state_capacity_dataset.meta.yml

```diff
@@ -0,0 +1,447 @@
+      weberian:
+        title: Index of Weberianness (Rauch and Evans 2000)
+        unit: ""
+        short_unit: ""
+        description_short: Measures the degree to which state agencies involved with economic policy exhibit Weberian characteristics. It ranges from 0 to 1 (greater Weberian characteristics).
+        description_from_producer: >-
+          An index created from expert ratings on the degree to which state agencies involved with economic
+          policy exhibit Weberian characteristics: degree of hiring through meritocratic processes, longevity
+          of employment in agencies, prospects for promotion within agencies, degree to which agency
+          staff remain in public service rather than move in and out of public sector, salary parity with
+          private sector for positions requiring comparable training, proportion of earnings coming through
+          salary rather than bribe collection, and the availability of non-public sector employment options
+          for graduates of elite universities. Country experts were asked to rate the years 1970 to 1990. We
+          use data for 34 countries.
+        processing_level: minor
+        display:
+          name: Index of Weberiannes
+          numDecimalPlaces: 1
+      wbstat:
+        title: Statistical Capacity (World Bank)
+        unit: ""
+        short_unit: ""
+        description_short: Measures the capacity of a country to collect and process statistical data. It ranges from 0 to 1 (greater statistical capacity).
+        description_from_producer: >-
+          In the original, the World Bank’s Statistical Capacity Indicator is a scale that ranges from 0 to 100.
+          It is based on a set of 25 criteria that fall into three main categories: methodology, sources of data,
+          and periodicity/timeliness of data:
+          - Methodology includes: balance of payments manual in use, CPI base year, government finance accounting, external debt reporting, import/export price indices, industrial production
+          index, national accounts base year, accuracy of data on national immunization coverage,
+          special dissemination standard, and UNESCO reporting.
+          - Sources include: agricultural census, health survey, population census, poverty survey, and
+          vital registration system coverage.
+          - Periodicity/timeliness measures reporting of data on: access to water, child malnutrition,
+          child mortality, gender equality, HIV/AIDS, immunizations, income poverty, maternal health,
+          per capita GDP growth, and primary school completion.
+
+
+          We have divided the resulting measure by 100 so that 1 is the highest possible value. The data
+          cover 127 countries from 2004 to 2015.
+        processing_level: minor
+        display:
+          name: Statistical Capacity
+          numDecimalPlaces: 1
+      tax_inc_tax:
+        title: Taxes on income, profits and capital gains (% of taxes)
```

Comment:
Instead of "(% of taxes)" you could say "as a share of total tax revenue". Idem for other similar indicator titles.